client/my-sites/sidebar-unified/static-data/fallback-menu.js

```diff
@@ -71,17 +71,24 @@ export default function buildFallbackResponse( {
 				},
 				{
 					parent: 'upgrades',
-					slug: 'Purchases',
-					title: translate( 'Purchases' ),
+					slug: 'Domains',
+					title: translate( 'Domains' ),
 					type: 'submenu-item',
-					url: `/purchases/subscriptions/${ siteDomain }`,
+					url: `/domains/manage/${ siteDomain }`,
 				},
 				{
 					parent: 'upgrades',
-					slug: 'Domains',
-					title: translate( 'Domains' ),
+					slug: 'Emails',
+					title: translate( 'Emails' ),
 					type: 'submenu-item',
-					url: `/domains/manage/${ siteDomain }`,
+					url: `/email/${ siteDomain }`,
```

Comment:
It would be great to switch to `/emails` now that we're introducing this new menu. We can set up a redirect for `/email`.